src/components/PoolDashboard/PoolDashboard.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import {useAsync} from 'react-use';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import { useQuery } from '@apollo/react-hooks';
+import gql from 'graphql-tag';
+import { fromWei } from 'web3-utils'
+import get from 'lodash/get'
+
+import { getCurrentPrize, fetchPoolMetadata } from '../../ethereum/pool';
+import MainButton from '../UI/MainButton';
+
+const GET_POOL = gql`
+  query Pool($poolAddress: String!) {
+    funding(id: $poolAddress) {
+      address,
+      totalStaked,
+      numberOfPlayers
+    }
+  }
+`;
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'column'
+  },
+  title: {
+    fontSize: '2.25em',
+    fontWeight: 'bold',
+    lineHeight: '2em',
+    paddingBottom: '0.2em',
+  },
+  desc: {
+    width: '550px',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    color: theme.customColors.text,
+  },
+  bar: {
+    display: 'flex',
+    padding: '30px',
+    '&> div': {
+      padding: '10px'
+    },
+  },
+  barTitle: {
+    color: theme.customColors.text,
+    padding: '5px',
+  },
+  barValue: {
+    fontSize: '1.5em',
+    fontWeight: 'bold',
+    padding: '5px',
+  },
+  cover: {
+    width: '100%',
+    height: '300px',
+    position: 'relative',
+    overflow: 'hidden',
+    '&> img': {
+      position: 'absolute',
+      top: '-300px',
+      width: '100%'
+    }
+  }
+}));
+
+const PoolDashboard = ({ match: { params: { poolAddress }}}) => {
```

Comment:
can you explain this: "{ match: { params: { poolAddress }}}"
(im used to components getting 'props' param, but this one confuses me)

Reply:
yeah it's like:
`poolAddress = props.match.params.poolAddress` :)